content/browser/ml/webnn/dml/execution_context.h

```diff
@@ -64,7 +68,7 @@ class ExecutionContext final : public base::RefCounted<ExecutionContext> {
   ComPtr<ID3D12Device> d3d12_device_;
   // There is one active command recorder at a time.
   CommandRecorder command_recorder_;
-  scoped_refptr<CommandQueue> command_queue_;
+  CommandQueue* command_queue_;
```

Comment:
```suggestion
  CommandQueue* command_queue_ = nullptr;
```

Now that this is a raw weak pointer, the lifetime of ownership is unclear. Adding a comment about who (what class) owns this would be useful for later readers.